package.json

```diff
@@ -72,6 +72,9 @@
     "build-scripts": "npm run build-script-main && npm run build-script-addons",
     "build-script-main": "browserify src/bundle.js --debug -o dist/lumagl.js && cat dist/lumagl.js | uglifyjs > dist/lumagl.min.js",
     "build-script-addons": "browserify src/addons/index.js -o dist/lumagl-addons.js --debug | uglifyjs > dist/lumagl-addons.min.js",
+    "watch-scripts": "npm run watch-script-main & npm run watch-script-addons",
+    "watch-script-main": "watchify src/bundle.js -v --debug -o dist/lumagl.js",
+    "watch-script-addons": "watchify src/addons/index.js -v --debug -o dist/lumagl-addons.js",
```

Comment:
I added these in lieu of fixing the budo script, not sure if/how that can be done.